src/app/Instance/DeleteInstance/DeleteInstance.tsx

```diff
@@ -54,15 +54,22 @@ const DeleteInstance = (props: DeleteInstanceProps): JSX.Element => {
       setPreloading(false);
       if (processorListResponse.total && processorListResponse.total > 0) {
         setDeleteBlockedReason(
-          t("instance.errors.cantDeleteBecauseProcessorsInside")
+          <Trans
```

Comment:
Not great to store jsx inside a state variable. Can you do this in other ways?

Reply:
Just a brief overview to justify my choice: the delete blocked reason message can be:
1) a message (already translated) returned by API
2) a message w/ interpolation computed on the UI
3) a message w/o interpolation computed on the UI
One concept was starting to be represented by multiple states and lead to complex management (on the `DeleteModal` side).

BTW, I agree with you that it is not recommended to have a JSX as a state, so the solution I can propose to you is to have a string already translated but with potential formatting to be applied (e.g. `<strong>`) on the `DeleteModal`, so I introduced (https://github.com/5733d9e2be6485d52ffa08870cabdee0/sandbox-ui/pull/106/commits/48ac1dfe0c1167184728ffd97e58bf764242a88e) a library to translate a string, with HTML tags, to a JSX element.

Please let me know your opinion.

Reply:
I see you also did an attempt with the `Trans` component to parse the translated string. 
Any drawbacks using it? It seems to do the job.
it could spare us the addition of another dependency just for this.

Reply:
It was working (I checked its source code, and behind the scenes, it was using a parsing library, too), with a warning (repeated on every render) of a missing translation key. 
It is probably designed only for translation purposes, so I stopped using it. 
In our case, we need to parse a string, including HTML tags.

There is an alternative (https://codingshower.com/parse-and-convert-raw-html-string-into-react-elements-or-components/), represented by using `dangerouslySetInnerHTML` feature, but it does not automatically sanitize the input, so I thought that the introduction of a library was the best option.

Reply:
Yes, we need to add another library: DOMPurify.
Before adding it, let's decide how we want to proceed.

Reply:
We will temporarily use the `Trans` component as agreed in a separate discussion.
Since use-cases are increasingly complex, it is acceptable to review and re-design components, particularly the communication between them. This will be done as part of https://issues.redhat.com/browse/MGDOBR-964